tools/nnyeah/nnyeah/NStrings.Designer.cs

```diff
@@ -0,0 +1,116 @@
+﻿//------------------------------------------------------------------------------
+// <auto-generated>
+//     This code was generated by a tool.
+//     Runtime Version:4.0.30319.42000
+//
+//     Changes to this file may cause incorrect behavior and will be lost if
+//     the code is regenerated.
+// </auto-generated>
+//------------------------------------------------------------------------------
+
+namespace nnyeah {
+    using System;
+    using System.Reflection;
+    
+    
+    [System.CodeDom.Compiler.GeneratedCodeAttribute("System.Resources.Tools.StronglyTypedResourceBuilder", "4.0.0.0")]
+    [System.Diagnostics.DebuggerNonUserCodeAttribute()]
+    [System.Runtime.CompilerServices.CompilerGeneratedAttribute()]
+    internal class NStrings {
```

Comment:
As @mandel-macaque already pointed out I think we should be consistent on naming, also on our project context `NStrings` is very very close to other names we use for other things that are pretty much unrelated to this so it would be really open for confusion.

Reply:
Changed to Errors.resx!